File: SIMPLEAPI_Demo/SIMPLEAPI_Demo/Clases/CertificadoDigital.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SIMPLEAPI_Demo.Clases
{
    public class CertificadoDigital
    {
        public string Nombre { get; set; }
        public string Rut { get; set; }

        public int RutCuerpo { get { return int.Parse(Rut.Substring(0, Rut.Length - 2)); } }
        public string DV { get { return Rut.Substring(Rut.Length - 1, 1); } }
    }
}

File: SIMPLEAPI_Demo/SIMPLEAPI_Demo_VB/Clases/CertificadoDigital.vb
Public Class CertificadoDigital
    Public Property Nombre As String
    Public Property Rut As String

    Public ReadOnly Property RutCuerpo As Integer
        Get
            Return Integer.Parse(Rut.Substring(0, Rut.Length - 2))
        End Get
    End Property

    Public ReadOnly Property DV As String
        Get
            Return Rut.Substring(Rut.Length - 1, 1)
        End Get
    End Property

End Class
